.github/ISSUE_TEMPLATE/epic-issue.md

```diff
@@ -0,0 +1,16 @@
+## High Level User Story
+As a _____, I need _____ so I can _____.
+
+## Hypothesis/(es)
+**If** _we make this change_ **then** _we expect this to happen_.
+
+## Goals
+_What outcome(s) do we want to see?_ 
+
+## KRs
+_Which VSP KRs does this body of work push forward?_ (`Intake & Onboarding`, `Automation + Instrumentation`, `Support`, `Capacity-building`)
```

Comment:
KRs or Objectives? I think KRs make more sense here but it seems like you listed Objectives as examples

Reply:
great, let's do that then?